src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -443,6 +443,12 @@ def set_policy(client, resource_group_name, vault_name, policy, policy_name):
     retention_range_in_days = policy_object.properties.instant_rp_retention_range_in_days
     schedule_run_frequency = policy_object.properties.schedule_policy.schedule_run_frequency
 
+    if schedule_run_frequency == 'Hourly':
+        raise InvalidArgumentValueError(
+            """
+            Hourly IaasVM backups are not allowed from CLI currently. Please try using portal experience for the same.
+            """)
+
     # Validating range of days input
     if schedule_run_frequency == 'Weekly' and retention_range_in_days != 5:
```

Comment:
> if schedule_run_frequency == 'Hourly':
> if schedule_run_frequency == 'Weekly'

Could we define enumerations or constants for the values of these variables to facilitate reuse?